obeam.opam

```diff
@@ -11,6 +11,7 @@ build: [
   ["dune" "build" "-p" name "-j" jobs]
 ]
 depends: [
+  "ocaml"                      {>= "4.07.0"}
```

Comment:
Is it OK to support only 4.07.0, which is the version used in CI?

Reply:
:+1: I will change this constraints in another PR if there are some requests to support older versions.

Reply:
OK! thanks